pandas/tests/plotting/frame/test_frame_color.py

```diff
@@ -151,14 +160,19 @@ def test_bar_user_colors(self):
         )
         # This should *only* work when `y` is specified, else
         # we use one color per column
-        ax = df.plot.bar(y="A", color=df["color"])
-        result = [p.get_facecolor() for p in ax.patches]
         expected = [
             (1.0, 0.0, 0.0, 1.0),
             (0.0, 0.0, 1.0, 1.0),
             (0.0, 0.0, 1.0, 1.0),
             (1.0, 0.0, 0.0, 1.0),
         ]
+
+        ax = df.plot.bar(y="A", color=df["color"])
+        result = [p.get_facecolor() for p in ax.patches]
+        assert result == expected
+
+        ax = df.plot.bar(y="A", c=df["color"])
+        result = [p.get_facecolor() for p in ax.patches]
```

Comment:
```suggestion
        ax = df.plot.bar(y="A", **{kw: df["color"]})
        result = [p.get_facecolor() for p in ax.patches]
```

and you can use 

```
@pytest.mark.parametrize("kw", ["color", "c"])
def test_bar_user_colors(self, kw):
````

to avoid code duplication.